commands/domains.go

```diff
@@ -38,8 +38,8 @@ func Domain() *Command {
 
 	cmdDomainCreate := CmdBuilder(cmd, RunDomainCreate, "create <domain>", "create domain", Writer,
 		aliasOpt("c"), displayerType(&domain{}), docCategories("domain"))
-	AddStringFlag(cmdDomainCreate, doctl.ArgIPAddress, "", "IP address", requiredOpt())
-
+	AddStringFlag(cmdDomainCreate, doctl.ArgIPAddress, "", "", "IP address", requiredOpt())
+	
```

Comment:
@nanzhong fixed. :D I hope it's correct now. Also fixed indentation for `volumes.go` at one point.
And yea, commit should be formatting but whatever. :P